dev-vcs/git-publish/git-publish-1.7.0.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..9} )
+inherit python-single-r1 python-utils-r1
```

Comment:
Do not inherit `python-utils-r1.eclass` directly.
`python-single-r1.eclass` already inherits `python-utils-r1.eclass`.